jupyter-lab/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM jupyter/minimal-notebook:latest
+LABEL org.opencontainers.image.source https://github.com/scilifelabdatacentre/stackn-images
+
+USER root
+RUN /bin/bash -c "apt update"
+RUN /bin/bash -c "apt install curl -y"
+COPY cli /home/jovyan/cli
+COPY examples /home/jovyan/work/examples
```

Comment:
Now we need these folders when deploying. I think we should update this so it pulls the folder from the git repo. What do you think?

Reply:
Yes, agree.